blockhead.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: blockhead
+
+addons:
+- name: bpm
+  jobs:
+  - name: bpm
+    release: bpm
+
+instance_groups:
+- name: blockhead-broker
```

Comment:
let's rename this to `broker`. the deployment name is good enough for the qualifier